src/libzstd.jl

```diff
@@ -163,6 +163,56 @@ end
 const ZSTD_CONTENTSIZE_UNKNOWN = Culonglong(0) - 1
 const ZSTD_CONTENTSIZE_ERROR   = Culonglong(0) - 2
 
-function find_decompressed_size(src::Ptr, size::Integer)
-    return LibZstd.ZSTD_findDecompressedSize(src, size)
+"""
+    find_decompressed_size(src::Vector{UInt8})
+    find_decompressed_size(src::Ptr, srcSize::Integer)
+
+Find the decompressed size of a source buffer containing one or more frames.
+
+This function should act identically to `ZSTD_findFrameDecompressedSize` which
+is part of the static only API.
+
+Normally, this function should return the size in terms of bytes of all the frames
+decompressed.
+
+May return `ZSTD_CONTENTSIZE_UNKNOWN` or `ZSTD_CONTENTSIZE_ERROR`:
+1. Return a code above if `ZSTD_getFrameContentSize` returns the code.
+2. Return `ZSTD_CONTENTSIZE_ERROR` if `ZSTD_findFrameCompressedSize` errors.
+3. Return `ZSTD_CONTENTSIZE_ERROR` if the frame extends beyond `srcSize`
+"""
+function find_decompressed_size(src::Vector{UInt8})
+    GC.@preserve src find_decompressed_size(pointer(src), length(src))
+end
+function find_decompressed_size(src::Ptr, srcSize::Integer)
+    frameOffset = 0
+    decompressedSize = 0
```

Comment:
```suggestion
    frameOffset = Csize_t(0)
    decompressedSize = Culonglong(0)
```

To ensure the `+=`s later on are type stable